about/faq.rst

```diff
@@ -391,6 +395,26 @@ engine instead. Keep in mind rendering engines usually have smaller communities
 compared to Godot. This will make it more difficult to find answers to your
 questions.
 
+What user interface toolkit does Godot use?
```

Comment:
Is Godot's UI system considered to be *immediate mode*, *retained mode* or something in between? This would be nice to mention while we're at it.

Reply:
I don't know, but let's merge anyway as the added section is good.